### DISCUSSION TITLE
Add a separate `DD+ Atmos` and a `TrueHD Atmos` overlay

### DISCUSSION THREAD

**Discussion by TRaSH-** [meisnate12/Plex-Meta-Manager-Configs#363] [Ideas]
I was looking at the options for overlays, and i noticed i'm missing a `DD+ Atmos` and a `TrueHD Atmos` overlay
Because the main audio makes a huge difference in my opinion, especially if you got a setup that supports it.

**Comment by FonduemangVI** [comment]
I posted in disc about this but agreed would be good to have separate overlays for these
